content/en/project/reviving.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: Reviving Medley
+weight: 80
+type: docs
+---
+# Choices and Challenges 
+Instead of starting with an earlier, finished and tested release, we started with a project that was valuable but incomplete: the files for a "Medley 3.5" release, as the latest available to us. Among other things, Medley 3.5 increase in address space -- from 64mb to 256mb.
+
+Differences between Medley 1.0 and Medley Interlisp.
```

Comment:
Does something more compact like this work better?

|   | Medley 1.0 | Medley 3.5 | Medley Interlisp |
| -- | --------- | ------------- | -----------------|
| CPU and OS | 16 and 32 bit, big endian | 32 bit little endian | 64 bit data path, big endian |
| File System | case insensitive, versioned | case insensitive, versioned | case sensitive, versioned |
| Mouse | three button | three button |  one, two, or touchpad cursor movement, scroll wheel |

Reply:
i wanted to leave room for saying more on each, as we make progress. 
Rather than make it more compact, I changed the areas of work into section heads.